cmd/monaco/integrationtest/assert.go

```diff
@@ -234,6 +242,35 @@ func AssertAutomation(t *testing.T, c automation.Client, env manifest.Environmen
 	}
 }
 
+func AssertBucket(t *testing.T, client bucket.Client, env manifest.EnvironmentDefinition, available bool, cfg config.Config) {
+
+	var expectedId string
+	if cfg.OriginObjectId != "" {
+		expectedId = cfg.OriginObjectId
+	} else {
+		expectedId = deploy.BucketId(cfg.Coordinate)
+	}
+
+	_, err := client.Get(context.TODO(), expectedId)
+	if err != nil && !strings.Contains(err.Error(), "404") {
```

Comment:
This method returns a ResponseError doesn't it? 
Not 100% necessary for a testing util, but I think it'd be cleaner to errors.As it to a Response error and look at the actual status code for the checks.